pystac_client/client.py

```diff
@@ -42,6 +43,9 @@
     from pystac.item import Item as Item_Type
 
 
+Timeout: TypeAlias = Optional[Union[float, Tuple[float, float], Tuple[float, None]]]
```

Comment:
This needs to be used in `stac_api_io.py` as well, correct?

Reply:
Whoops!  Totally.